client/ayon_core/hosts/substancepainter/plugins/load/load_mesh.py

```diff
@@ -8,10 +9,148 @@
+    """The pop-up dialog allows users to choose material
+    duplicate options for importing Max objects when updating
+    or switching assets.
+    """
+    def __init__(self, project_templates):
+        super(SubstanceProjectConfigurationWindow, self).__init__()
+        self.setWindowFlags(self.windowFlags() | QtCore.Qt.FramelessWindowHint)
+
+        self.import_cameras = False
+        self.preserve_strokes = False
+        self.template_name = None
+        self.template_names = [template["name"] for template
+                               in project_templates]
+        self.project_templates = project_templates
+
+        self.widgets = {
+            "label": QtWidgets.QLabel(
+                "Select your template for project configuration"),
+            "template_options": QtWidgets.QComboBox(),
+            "import_cameras": QtWidgets.QCheckBox("Import Cameras"),
+            "preserve_strokes": QtWidgets.QCheckBox("Preserve Strokes"),
+            "clickbox": QtWidgets.QWidget(),
+            "combobox": QtWidgets.QWidget(),
+            "buttons": QtWidgets.QDialogButtonBox(
+                QtWidgets.QDialogButtonBox.Ok
+                | QtWidgets.QDialogButtonBox.Cancel)
+        }
+        for template in self.template_names:
+            self.widgets["template_options"].addItem(template)
+
+        template_name = self.widgets["template_options"].currentText()
+        self.get_boolean_setting(template_name)
+        # Build clickboxes
+        layout = QtWidgets.QHBoxLayout(self.widgets["clickbox"])
+        layout.addWidget(self.widgets["import_cameras"])
+        layout.addWidget(self.widgets["preserve_strokes"])
+        # Build combobox
+        layout = QtWidgets.QHBoxLayout(self.widgets["combobox"])
+        layout.addWidget(self.widgets["template_options"])
+
+        # Build buttons
+        layout = QtWidgets.QHBoxLayout(self.widgets["buttons"])
+        # Build layout.
+        layout = QtWidgets.QVBoxLayout(self)
+        layout.addWidget(self.widgets["label"])
+        layout.addWidget(self.widgets["combobox"])
+        layout.addWidget(self.widgets["clickbox"])
+        layout.addWidget(self.widgets["buttons"])
+
+        self.widgets["template_options"].currentTextChanged.connect(
+            self.on_options_changed)
+        self.widgets["buttons"].accepted.connect(self.on_ok_pressed)
+        self.widgets["buttons"].rejected.connect(self.on_cancel_pressed)
+
+    def on_options_changed(self, value):
+        self.get_boolean_setting(value)
+
+    def on_ok_pressed(self):
+        if self.widgets["import_cameras"].isChecked():
+            self.import_cameras = True
+        if self.widgets["preserve_strokes"].isChecked():
+            self.preserve_strokes = True
+        self.template_name = (
+            self.widgets["template_options"].currentText()
+        )
+        self.close()
+
+    def on_cancel_pressed(self):
+        self.template_name = None
```

Comment:
We don't need to set it to `None` because it should be `None` by default - we are only setting a value to it if the user accepts the dialog.
```suggestion
```

Reply:
i would remove this in the upcoming update.